kaiax/gov/headergov/impl/schema.go

```diff
@@ -10,15 +12,14 @@ import (
 var (
 	voteDataBlockNumsKey = []byte("governanceVoteDataBlockNums")
 	govDataBlockNumsKey  = []byte("governanceDataBlockNums")
+	lastInsertedBlockKey = []byte("governanceLastInsertedBlock") // grows downwards
```

Comment:
LowestBlockNums? LowestInsertedEpochBlock?

Reply:
In a below comment, Lake suggested `LastInsertedVoteAnchor`.

How about `LowestScannedAnchor`? I'm not sure if "anchor" is an appropriate term though.

Reply:
@blukat29 @hyunsooda Renamed to `LowestVoteScannedBlock`